src/components/Header/SearchBar/SearchBar.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import {render, screen} from '@testing-library/react';
+import SearchBar from './index';
+
+test('renders SearchBar', () => {
+    render(<SearchBar />);
+
+    const title = screen.getByPlaceholderText(/Search.../i);
```

Comment:
Using `data-testid` in your components will help you target that specific component in your test cases.